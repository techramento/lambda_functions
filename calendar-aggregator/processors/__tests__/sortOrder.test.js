const MockEvents = require('../../__fixtures__/events.json')
const sortOrder = require('../sortOrder')

describe('sortOrder processor', () => {
  it('sorts the events by their start date', () => {
    expect(sortOrder(MockEvents)).toMatchSnapshot()
  })
})
