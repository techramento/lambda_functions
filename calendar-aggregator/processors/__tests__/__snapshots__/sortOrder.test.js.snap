// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sortOrder processor sorts the events by their start date 1`] = `
Array [
  Object {
    "start": Object {
      "dateTime": "2017-01-01T00:00:00Z",
    },
  },
  Object {
    "start": Object {
      "dateTime": "2017-02-01T11:59:59Z",
    },
  },
  Object {
    "start": Object {
      "dateTime": "2017-02-01T11:59:59Z",
    },
  },
  Object {
    "start": Object {
      "dateTime": "2017-12-31T11:59:59Z",
    },
  },
]
`;
